src/pixie/fonts.nim

```diff
@@ -450,14 +503,46 @@ proc strokeText*(
   dashes: seq[float32] = @[]
 ) =
   ## Strokes the text arrangement.
+  var line: int
   for spanIndex, (start, stop) in arrangement.spans:
-    let font = arrangement.fonts[spanIndex]
+    let
+      font = arrangement.fonts[spanIndex]
```

Comment:
Duplicated code from fill?

Reply:
It's duplicated yes. I'm not finding a good way to remove it though. Maybe you have an idea